src/hooks/job-types/use-job-type-parameters.js

```diff
@@ -0,0 +1,22 @@
+import useJobType from './use-job-type'
+
+const useJobTypeParameters = (jobType) => {
+    const fetch = useJobType(jobType)
+
+    // Return parameters when there is data
+    if (fetch.data) {
+        const hasParameters = 'jobParameters' in fetch.data
+
+        if (!hasParameters) {
+            return { ...fetch, data: [] }
+        }
+
+        const data = fetch.data.jobParameters
+
+        return { ...fetch, data }
```

Comment:
```suggestion
        const data = fetch.data.jobParameters
        return { ...fetch, data: data || [] }
```

Reply:
Fixed in [0f62c1a](https://github.com/dhis2/scheduler-app/pull/495/commits/0f62c1a4f3b8462dab031f580ec507d772950367)